lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntime.cpp

```diff
@@ -118,8 +118,11 @@ enum class RuntimeKind { Swift, ObjC };
 
 /// Detect a statically linked Swift runtime by looking for a well-known symbol.
 static bool IsStaticSwiftRuntime(Module &image) {
-  static ConstString swift_release_dealloc_sym("_swift_release_dealloc");
-  return image.FindFirstSymbolWithNameAndType(swift_release_dealloc_sym);
+  ConstString swift_release_dealloc_sym("_swift_release_dealloc");
+  if (image.FindFirstSymbolWithNameAndType(swift_release_dealloc_sym))
+    return true;
+  ConstString swift_release_sym("swift_release");
+  return image.FindFirstSymbolWithNameAndType(swift_release_sym);
```

Comment:
Can you add a comment that the second case is for embedded Swift and the first one is for statically linked Swift? Also, does the second case work for both?

Reply:
I changed it to `__swift_reflection_version", which works for both, even for the simplest program after dead stripping.